os/services/lwm2m/lwm2m-notification-queue.c

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (c) 2017, RISE SICS AB.
```

Comment:
Are we assuming that you are a member of the SICS team? If not, then this perhaps needs to change

Reply:
Yes, I've been working in the NES group as a Master student since September.

Reply:
👍